docs/api/dengue_nowcast.md

```diff
@@ -0,0 +1,72 @@
+# About
+
+This is the documentation of the API for accessing the Delphi's Dengue Nowcast (`dengue_nowcast`) data source of
+the [Delphi](https://delphi.cmu.edu/)'s epidemiological data.
+
+## Delphi's Dengue Nowcast Data
+
+... <!-- TODO -->
+
+## Contributing
+
+If you are interested in contributing:
+
+- For development of the API itself, see the
+  [development guide](docs/epidata_development.md).
+- To suggest changes, additions, or other ways to improve,
+  [open an issue](https://github.com/cmu-delphi/delphi-epidata/issues/new)
+  describing your idea.
+
+## Citing
+
+We hope that this API is useful to others outside of our group, especially for
+epidemiological and other scientific research. If you use this API and would
+like to cite it, we would gratefully recommend the following copy:
+
+> David C. Farrow,
+> Logan C. Brooks,
+> Aaron Rumack,
+> Ryan J. Tibshirani,
+> Roni Rosenfeld
+> (2015).
+> _Delphi Epidata API_.
+> https://github.com/cmu-delphi/delphi-epidata
+
+## Data licensing
+
+Any data which is produced novelly by Delphi and is intentionally and openly
+surfaced by Delphi through this API is hereby licensed
+[CC-BY](https://creativecommons.org/licenses/by/4.0/). The `delphi` endpoint is known to wholly or partially serve data
```

Comment:
looks like copypasta from `delphi` endpoint, recommend removing

Reply:
replaced with the non-delphi licensing stub i use elsewhere.

Reply:
actually, the reason i concluded this was a delphi set and used the delphi licensing wording (albeit with the incorrect source name) was because in places like the following it is referred to as delphi's:
https://github.com/cmu-delphi/delphi-epidata/blob/c48af8f126e935306eaebb79fcf1b22a201a3dfa/src/client/delphi_epidata.py#L475

similar for the dengue_sensors set. was this an incorrect conclusion?